frontend/apps/marketing/src/components/contentful/collections/peopleCollection/PeopleCollection.tsx

```diff
@@ -0,0 +1,164 @@
+import OpenInNew from '@mui/icons-material/OpenInNew';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import {EntryFields} from 'contentful';
+import {useMemo, useId} from 'react';
+
+import {getAbsoluteImageUrl} from '@/selectors/contentful/getImage';
+import {LinkEntry} from '@/types/contentful/entries/Link';
+import {Entry} from '@/types/contentful/Entry';
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+type ItemFields = {
+  name: EntryFields.Text;
+  title: EntryFields.Text;
+  image?: ExperienceAsset;
+  bio?: EntryFields.Text;
+  personalLink?: LinkEntry;
+};
+
+type ItemEntry = Entry<ItemFields>;
+
+export type PeopleCollectionProps = {
+  /** Collection content w/ fields from Contentful */
+  people: ItemEntry[];
+  /** Show or hide images */
+  imageVisibility?: 'show' | 'hide';
+};
+
+const styles = {
+  container: {
+    alignItems: 'flex-start',
+  },
+  gridItem: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    textAlign: 'center',
+  },
+  image: {
+    marginBottom: '1.25rem',
+    width: 128,
+    height: 128,
+    borderRadius: '50%',
+  },
+  overline: {
+    color: '#4C5661',
```

Comment:
```suggestion
    color: var(---neutral-gray-90),
```

Or perhaps a more idiomatic MUI approach is to define a pallete color under `theme.vars.palette.primary` for this (but we can wait on that for later).